src/Support/ModularizedCommandsServiceProvider.php

```diff
@@ -30,6 +31,7 @@
 class ModularizedCommandsServiceProvider extends ServiceProvider
 {
 	protected $overrides = [
+		'command.livewire.make' => MakeLivewire::class,
```

Comment:
It doesn't appear that Livewire registers the "command.livewire.make" singleton anywhere. I you [look at the source code][1], it seems like it's just using the standard command registration. That makes me think that this will only work, then, if the modular service provider happens to run **after** the livewire service provider. Seems like that's not ideal.

[1]: https://github.com/livewire/livewire/blob/master/src/LivewireServiceProvider.php#L153

Reply:
Yeah me and @DanielCoulbourne was thinkering with this to make it work and we what we could find was that Livewire was hooking the make command in a not standard way so we didn’t find a solution for adding the —module argument into LW.